scripts/copy-declarations.js

```diff
@@ -0,0 +1,31 @@
+// Copy manually-written `.d.ts` files from `src/` to `types/src/`
+import { copyFileSync } from "node:fs";
+import { dirname, resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+
+const SOURCE = "../src/";
+const DEST = "../types/src/";
+
+/**
+ * `.d.ts` files to copy.
+ * Paths written relative to the `src/` directory
+ */
+const TO_COPY = [
+	"color.d.ts",
+	"ColorSpace.d.ts",
+	"hooks.d.ts",
+	"index.d.ts",
+	"space-coord-accessors.d.ts",
+	"space.d.ts",
+	"types.d.ts",
+];
+
+/** Script file directory */
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
```

Comment:
Create the destination directory if it doesn't exist. This is needed if we swap the order of operations in `build:ts`.

```suggestion
mkdirSync(resolve(__dirname, DEST), { recursive: true });
```